lib/kramdown-rfc2629.rb

```diff
@@ -738,7 +760,12 @@ def convert_xref(el, indent, opts)
           else
             gi ||= "xref"
           end
-          "<#{gi}#{el_html_attributes(el)}/>"
+          if text
+            tail = ">#{escape_html(text, :text)}</#{gi}>"
```

Comment:
That is the part that needs to do the recursive kramdown call...

cf. lib/kramdown-rfc/parameterset.rb line 38+

Reply:
I just factored out a `::Kramdown::Converter::Rfc2629::process_markdown`... b94fb8c